src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-rpm-template

```diff
@@ -38,7 +38,7 @@
 
 exec="${{chdir}}/bin/${{exec}}"
 prog="${{app_name}}"
-lockfile=/var/lock/subsys/$prog
+lockfile="/var/lock/subsys/${{app_name}}"
```

Comment:
Yeah, if it detects `$…` then it issues a warning as it can’t resolve the substitution. Otherwise, it’s an error.